query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/relations/compound_fks_mixed_requiredness.rs

```diff
@@ -49,12 +49,17 @@ mod compound_fks {
           @r###"{"data":{"createOnePost":{"id":1,"user_id":1,"user_age":null,"User":null}}}"###
         );
 
-        // Foreign key violation
-        assert_error!(
-            &runner,
-            r#"mutation{createOnePost(data:{id: 2, user_id:2, user_age: 2}){id, user_id, user_age, User{id}}}"#,
-            2003
-        );
+        // Foreign key violation, which doesn't happen on PlanetScale.
```

Comment:
Why though?

Reply:
Because foreign keys are disabled by default on PlanetScale

Reply:
Could we add this to the comment?
Also, shouldn't we have an emulated error in this case?